notes/stories/television/tv-theamericanshortstory.st.txt

```diff
@@ -406,3 +405,52 @@ fear [Bill Kenner admitted to Harold that "he was scared" fighting in the Great
+old-age love [Longtime married couple Lucy and Charley quarreled on their golden wedding anniversary holiday, but in the end they came to love one another more than ever. Lucy and Charley spoke often of their many years together as a couple.]
+reunion with a loved one [Central to story was Lucy running into her ex-fiancé Frank while she and her husband were vacationing in Florida.]
+old flames [Central to story was Lucy running into her ex-fiancé Frank while she and her husband were vacationing in Florida.]
+romantic jealousy [Charley was jealous of Lucy's ex-fiancé, Frank. Cora was jealous of Lucy when Charley brought up Lucy and Frank's previous betrothal.]
+coping with a jealous lover [From Lucy's point of view Charley was letting his jealousy of Frank spoil their golden wedding anniversary vacation in Florida. Lucy often chastised Charley for attacking Frank.]
+reconciliation [The story concluded with Charley and Lucy apologizing for their parts in almost ruining their golden wedding anniversary and embraced, laughing.]
+inferiority complex [Charley was self-conscious on account that his wife's ex-lover had become more wealthy than he; to wit, Charley repeatedly made derogatory comments about Frank's being a veterinarian. Charley's insecurity was further exacerbated by Lucy wanting to spend a lot of time with Frank and Cora Hartsell.]
+remembering bygone days [Lucy and Frank, who were now elderly and married to other people, reflected fondly on the old days when they were young and in love.]
+
+:: Minor Themes
+coping with an obnoxious chatterbox [Charley was several times chastised for talking too much to those around him. Charley grumbled about Cora being a chatterbox.]
+tabletop games [Charley delighted in slaughtering his perceived rival Frank at checkers. The Tates played cards with the Hartsells.]
+coping with senility [Charlie exhibited certain signs of senility, most notably when he almost missed the train while engaged in a jocular one-sided conversation and the many times he expressed his opinion without a filter.]
+dancing [The Tates and Hartsells went out for a night of dancing.]
+attitude of superiority [Cora and Frank made clear that they thought themselves superior to Lucy and Charley in terms of wealth and social status.]
+ventriloquism [A ventriloquist with dummy in hand cracked some politically oriented joked on stage at the social.]
+coping with humiliation [Lucy was visibly mortified when Charley stepped out onto the stage at the social and began telling embarrassing stories. Cora was mortified when her false teeth fell out onto the ground. Frank began sweating profusely after losing to Charley many times at checkers. Charley feigned a thumb injury to avoid losing to Frank at horseshoes.]
+sports [The Hartsells took the Tates out to play shuffleboard. Charley tried to beat Frank at horseshoes but failed miserably.]
```

Comment:
Lol, is there a word for such half-sport half-game activities?

Reply:
If only...